packages/grid/data-grid/rollup.config.js

```diff
@@ -44,14 +44,11 @@ export default [
     output: [{ file: 'dist/data-grid.d.ts', format: 'es' }],
     plugins: [
       dts(),
-      command(
-        `cat ../x-grid-modules/dist/x-grid-modules.d.ts ./dist/data-grid.d.ts > ./dist/data-grid.all.d.ts`,
-        {
-          exitOnFail: true,
-          wait: true,
-        },
-      ),
       !production && sourceMaps(),
+      command([`rm -f ./dist/DataGrid*`, `rm -f ./dist/index.d.ts `], {
```

Comment:
Why aren't we using `dist/index.d.ts`? I don't understand why we have a custom step to build the TypeScript definitions. It seems to be handled by default in https://github.com/ezolenko/rollup-plugin-typescript2. I assume I miss something.

Reply:
because we have the dts plugin that optimize the d.ts files and bundle them into 1 file the data-grid.d.ts file